util/include/crypto.hpp

```diff
@@ -0,0 +1,94 @@
+// Concord
+//
+// Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the sub-component's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#pragma GCC diagnostic push
```

Comment:
It's a generic header file, it shouldn't include specific implementations.
please remove all the `#include <crypto/*` and `#include <openssl/*`

Reply:
Changes done https://github.com/arc-vmware/concord-bft/pull/12/commits/6aaca0e61da69ebbaa897378ee75083aff98d4e4